upcloud/service/fixtures/createmodifydeletenetwork.yaml

```diff
@@ -71,31 +77,98 @@ interactions:
             "ip_networks" : {
                "ip_network" : [
                   {
-                     "address" : "172.16.0.0/22",
+                     "address" : "172.17.0.0/22",
                      "dhcp" : "yes",
                      "dhcp_default_route" : "no",
                      "dhcp_dns" : [
-                        "172.16.0.10",
-                        "172.16.1.10"
+                        "172.17.0.10",
+                        "172.17.1.10"
                      ],
                      "family" : "IPv4",
-                     "gateway" : "172.16.0.1"
+                     "gateway" : "172.17.0.1"
```

Comment:
Had to change it unfortunately because my account has this ip-address "hard occupied" :D